receiver/otlpreceiver/otlphttp.go

```diff
@@ -57,7 +96,7 @@ func handleTraces(
 	}
 
 	// TODO: Pass response from grpc handler when otlpgrpc returns concrete type.
-	writeResponse(resp, contentType, http.StatusOK, &types.Empty{})
+	writeResponse(resp, contentType, http.StatusOK, emptyMessage)
```

Comment:
I was a bit confused by this. For clarity we should probably write proper `Export*ServiceResponse` protobuf message, which indeed is empty right now, but may not be empty in the future if the proto evolves.